From 16e3a1e5129b9c6cb10bf9bb7730ea25c595dba8 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Fri, 11 Sep 2015 16:02:49 +0800
Subject: [PATCH 0717/1691] MLK-11533 uapi: Add isl29023.h into Kbuild

commit 16e3a1e5129b9c6cb10bf9bb7730ea25c595dba8 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

export this for 'make headers_install'

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 include/uapi/linux/Kbuild | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/Kbuild b/include/uapi/linux/Kbuild
index 7f23dde..aaeaf76 100644
--- a/include/uapi/linux/Kbuild
+++ b/include/uapi/linux/Kbuild
@@ -237,6 +237,7 @@ endif
 
 header-y += hw_breakpoint.h
 header-y += ipu.h
+header-y += isl29023.h
 header-y += l2tp.h
 header-y += libc-compat.h
 header-y += limits.h
-- 
1.9.1

